iouring_test.go

```diff
@@ -0,0 +1,45 @@
+package iouring
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func testMany(nreqs uint, t *testing.T) {
```

Comment:
It would be more accurate to call it `testSubmitRequests`, don't you think?

And, For the test function, I think it would be better to put the t parameter first.
eg.
```go
func testSubmitRequests(t *testing.T, nreqs uint)
```